README.md

```diff
@@ -83,4 +83,11 @@ marked.parse("I :heart: marked! :tada:");
 | option | default | description |
 |--------|---------|-------------|
 | emojis | required | An object with keys as emoji name and values as emoji. The values are assumed to be image urls (as returned by Octokit) unless `unicode` option is `true`. Values can also be an object with a `url` property or `char` property to allow unicode and url emoji at the same time. |
```

Comment:
Perhaps the description of the `emojis` option could be clarified to explain how it works with the renderer option, since adding the renderer option opens up the possible values here.

An object with keys as emoji name and values as emoji. **If a `renderer` option is supplied, the values can be any custom 
value as appropriate to the renderer output. If a `renderer` option is not supplied,** values are assumed to be image urls (as returned by Octokit) unless `unicode` option is `true`. Values can also be an object with a `url` property or `char` property to allow unicode and url emoji at the same time.